packages/app/src/main/applicationManager.ts

```diff
@@ -34,6 +34,9 @@ export class ApplicationManager {
       width: 1280,
       webPreferences: {
         preload: MAIN_WINDOW_PRELOAD_WEBPACK_ENTRY,
+        // This disables ability to use NODE functions in the render process
+        // it's important for firebase to work
+        nodeIntegration: false,
```

Comment:
@JohannesMerkt  this disables NODEJS functionality in the render process but I think we don't have anything from nodejs in render files right? Can you confirm this? Thanks

Reply:
Yes we do not have anything nodejs specific in the renderers. Was it set to true earlier? or not set at all?

Reply:
The default is true - it allows some limited things like file system.  Awesome 👍